src/typeahead/typeahead-window.ts

```diff
@@ -37,8 +45,17 @@ export class NgbTypeaheadWindow {
    * A function used to format a given result before display. This function should return a formated string without any
    * HTML markup.
    */
-  @Input() formatter = (result) => { return `${result}`; };
+  @Input() formatter = toStringFormatter;
```

Comment:
> just curious, why replace this?

Oh, this is due to TS / linter. TS interprets `formatter = (result) => { return `${result}`; };` as a "method" member and our linting rules are forcing methods after regular fields. Put the 2 together and you have to have `formatter` as the last field (even after outputs!).

> maybe put it to utils, as we're using it elswhere?

Makes sense. Will move it in a separate PR.